frontend/sw360-portlet/src/main/resources/META-INF/resources/html/projects/includes/projects/clearingStatus.jsp

```diff
@@ -45,6 +45,9 @@
 
 <div class="tab-content" id="pills-clearingStatusTab">
     <div class="tab-pane fade show active" id="pills-treeView" role="tabpanel" aria-labelledby="pills-tree-tab">
+    <div class="float-right mx-2">
+        <input type="text" id="search_table" placeholder="Search...">
```

Comment:
Since `Search...` is a hardcoded values.
Kindly use language.properties for I18N

Would it be possible to use bootstrap class `form-control` like it is used in other places like clearing request etc.
So that textbox looks similar in entire sw360

Reply:
Used `language.properties` for Search placeholder, and `form-control` class for Search input.